content/tutorials/modeling_movement/GRASS_movement.qmd

```diff
@@ -0,0 +1,565 @@
+
+-   Imagine a stranded hiker NE of Flagstaff who has to walk to FMC.
+
+-   What path would take the least time?
+
+![](img_movement/GRASS_movement_16.webp)
+
+## Generating a least cost path
+
+To create a LCP in GRASS, we will use [r.path](https://grass.osgeo.org/grass-stable/manuals/r.path.html) (also under the Raster/Terrain analysis menu).
+
+:::::::: {.panel-tabset group="language"}
+
+#### GUI
+
+::::: grid
+::: g-col-6
+
+1.  Input the **directions map** (*FMC_directions*) we also made when we created the cumulative cost surface.\
+    ![](img_movement/GRASS_movement_17.webp)
+:::
+
+::: g-col-6
+2.  Input the **coordinates of the hiker** as the starting point for the LCP. (We could also use a pre-defined vector point).
+
+![](img_movement/GRASS_movement_18.webp)
+:::
+:::::
+
+:::: grid
+::: g-col-6
+3.  Specify the **name of the output vector path map** (*LCP_cumulative*).
+
+![](img_movement/GRASS_movement_19.webp)
+:::
+::::
+
+#### Command line
+
+```{python}
+r.path input=FMC_directions format=auto vector_path=LCP_cumulative start_coordinates=477476,13914951
+```
+
+#### Python
+
+```{python}
+gs.run_command("r.path",
+               input="FMC_directions",
+               format="auto",
+               vector_path="LCP_cumulative",
+               start_coordinates=[477476, 13914951])
+```
+::::::::
+
+## Least cost path generated
+
+-   Here is the LCP result.
+
+![](img_movement/GRASS_movement_20.webp)
+
+# Adding a friction map to movement costs
+
+## Overview
+
+-   A **friction map** can be used to incorporate other factors than just terrain into creating a cumulative cost surface and modeling movement
+
+-   The value of each cell in a friction map raster is the amount of walking time, in seconds/meter, *in addition to* the time needed because of terrain
+
+-   We can **reclassify** the *landuse* map to create a friction map of the amount of extra time it would take to walk through different kinds of land cover
+
+![Reclassification of *landuse* to show major land cover types](img_movement/GRASS_movement_21.webp)
+
+## Reclassifying *landuse* to create a friction map
+
+A standard walking speed across flat terrain is about 5 km/hr = 0.72 sec/m.
+
+We might then estimate that it would take an additional
+
+-   3 sec/m to walk through dense pinyon-juniper woodland
+
+-   1 sec/m to cross conifer forest
+
+-   2 sec/m to wander through urban Flagstaff
+
+-   5 sec/m to clamber over lava fields
+
+-   10 sec/m to try and cross water
+
+We can create this friction map by reclassifying the *landuse* map using [*r.reclass*](https://grass.osgeo.org/grass-stable/manuals/r.reclass.html) to assign new friction values to the existing land cover categories.  
+
+:::::: {.panel-tabset group="language"}
+
+## GUI
+
+The [*r.reclass*](https://grass.osgeo.org/grass-stable/manuals/r.reclass.html) tool is found under the Raster/Change category… menu.
```

Comment:
```suggestion
The [r.reclass](https://grass.osgeo.org/grass-stable/manuals/r.reclass.html) tool is found under the Raster/Change category… menu.
```

Reply:
Thanks for the suggestion. That is indeed probably a cleaner look. I thought that the rest of the bold and italics were consistent across different items but will check.